plugins/repository-s3/src/test/java/org/opensearch/repositories/s3/S3ServiceTests.java

```diff
@@ -83,4 +86,8 @@ public void testCachedClientsWithCredentialsAreReleased() {
         final S3ClientSettings clientSettingsReloaded = s3Service.settings(metadata1);
         assertNotSame(clientSettings, clientSettingsReloaded);
     }
+
+    private Path configPath() {
```

Comment:
Nitpick - consider moving this to `OpenSearchTestCase` so you don't have to duplicate it across the various tests

Reply:
Thanks @kartg , I thought about that, the config path is introduced in the `OpenSearchIntegTestCase`, adding that to `OpenSearchTestCase` would introduce some confusion I believe. From the other side, making the plugin tests to be integration tests seems like too much. So I added this small function, but I could make it a trait instead, will be cleaner.